server/index.js

```diff
@@ -0,0 +1,6 @@
+import app from './app';
+import config from './app/config';
+import logger from './app/util/logger';
+
+app.listen(config.port);
+logger.info(`⚡⚡  Listening on localhost:${config.port} ⚡⚡`);
```

Comment:
Copy/paste :smile:. They just help with finding when/where the server starts up.